config/webpack/loaders/assets.js

```diff
@@ -0,0 +1,12 @@
+const { env, publicPath } = require('../configuration.js')
+
+module.exports = {
```

Comment:
We had some problems here @alebian. Would you check this for me please?

Reply:
Ping here @alebian

Reply:
Ping @alebian

Reply:
I have the exact same content